Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Middlewares/Sha256TreeHashMiddleware.swift

```diff
@@ -25,13 +25,7 @@ public struct Sha256TreeHashMiddleware<OperationStackInput, OperationStackOutput
     ) async throws {
         switch request.body {
         case .data(let data):
```

Comment:
got some warning messages in here we should take care of

Reply:
Addressed warnings